build.gradle

```diff
@@ -47,7 +46,6 @@ tasks.withType(JavaCompile) {
 shadowJar {
     archiveClassifier.set("")
 
-    relocate "org.bstats", "me.clip.placeholderapi.metrics"
```

Comment:
This should stay relocated or it might conflict with any other plugin that uses bStats without relocation (or even with Spigot's bStats itself)

Reply:
Just now saw the Metrics clas.... WHY?
This is pointless and just too much to just fix a download issue that (so far) barely anyone had.